crates/sui-core/src/authority.rs

```diff
@@ -1697,7 +1699,16 @@ impl AuthorityState {
             })?;
 
         let layout = match request.object_format_options {
-            Some(format) => object.get_layout(format, epoch_store.module_cache().as_ref())?,
+            Some(format) => match object.data.try_as_move() {
```

Comment:
Nit. Use map on the `Option` instead of `match`?

Reply:
I'll think about it, thanks